pkg/monitors/output.go

```diff
@@ -84,6 +86,20 @@ func (mo *monitorOutput) preprocessDP(dp *datapoint.Datapoint) bool {
 		return false
 	}
 
+	for i := range mo.metricNameTransformations {
+		reWithRepl := mo.metricNameTransformations[i]
+		re := reWithRepl.Regexp
+		repl := reWithRepl.Replacement
+
+		// An optimization for simple regexps (i.e. ones with no special
+		// matching syntax)
+		if prefix, complete := re.LiteralPrefix(); complete && prefix == dp.Metric {
+			dp.Metric = repl
+			continue
+		}
+		dp.Metric = re.ReplaceAllString(dp.Metric, repl)
```

Comment:
What happens if this is the empty string?

Reply:
Then the metric name will be an empty string.